lib/sms-sender-ids.json

```diff
@@ -0,0 +1,5 @@
+[
+  [ "CA", "16474909977" ],
+  [ "GB", "Firefox" ],
+  [ "US", "15036789977" ]
```

Comment:
A brief note about these sender ids. As per #1645, there is some work to do before we can get shortcodes working for USA and Canada numbers. I opted to spin that out as a separate issue, so we could get this work in sooner for proper hands-on testing.

In the meantime, these are just some virtual numbers I bought via Nexmo.